server/command.go

```diff
@@ -90,6 +90,12 @@ func executeConnect(p *Plugin, c *plugin.Context, header *model.CommandArgs, arg
 	if len(args) != 0 {
 		return help()
 	}
+
+	_, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
```

Comment:
0/5 would suggest changing https://github.com/mattermost/mattermost-plugin-jira/blob/master/server/kv.go#L280 to return an ErrNotFound, then check for it and not override other error messages (can wrap them, though)